### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SweeeD1**
- [DEMO LINK] https://SweeeD1.github.io/layout_catalog/
- [TEST REPORT LINK] https://SweeeD1.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,11 +2,475 @@
...
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+                laptops & computers
```
Comment (volodymyr-soltys97) [inline src/index.html:58]:
```suggestion
              laptops & computers
```

File: src/index.html [hunk 2]
```diff
@@ -2,11 +2,475 @@
...
+
+              <p class="review review--position">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="price">
+              <p
+                class="price__text">
+                Price:
```
Comment (volodymyr-soltys97) [inline src/index.html:134]:
```suggestion
              <p class="price__text">
                Price:
```

File: src/index.html [hunk 3]
```diff
@@ -2,11 +2,475 @@
...
+              </p>
+              <p
+                class="price_number">
```
Comment (volodymyr-soltys97) [inline src/index.html:230]:
```suggestion
              <p class="price_number">
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,210 @@
...
+  font-weight: 700;
+  border-radius: 5px;
+  border: 1px solid var(--main-color);
+  background-color: var(--main-color);
+
+  &:hover {
+    background-color: white;
+    color: var(--main-color);
+  }
+
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:209]:
```suggestion
```

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,210 @@
...
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:187]:
```suggestion
```

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,210 @@
...
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+p {
+  margin: 0;
+}
+
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:106]: It is necessary to bring up the style reset, or transfer it to a separate file

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
test

**Pull request closed without merging**